common/src/main/java/com/microsoft/identity/common/internal/logging/CommonCoreLogger.java

```diff
@@ -0,0 +1,264 @@
+package com.microsoft.identity.common.internal.logging;
+
+import android.os.Build;
+import android.support.annotation.Nullable;
+import android.util.Log;
+
+import com.microsoft.identity.common.adal.error.ADALError;

```

Comment:
Because `ADALError` instances get turned into Strings before getting logged, what if we consider removing the dependency between `ADALError` and `CommonCoreLogger` so that we're free to remove/rethink the existence of `ADALError` later? I'm leaning towards keeping the logger implementation "pure" and having zero dependencies we don't absolutely _need_. Thoughts?

/cc @addev-ashish @shoatman

Reply:
i am removing ADAL error, this code will break. i think this entire dependency is backwards